endpoints-control/src/main/java/com/google/api/control/Client.java

```diff
@@ -363,6 +398,51 @@ public void run() {
     }, interval, 1 /* not so high priority */);
   }
 
+  private void flushAndScheduleQuota() {
+    if (resetIfStopped()) {
+      log.log(Level.FINE, "did not schedule quota flush: client is stopped");
+      return;
+    }
+    int interval = quotaAggregator.getFlushIntervalMillis();
+    if (interval < 0) {
```

Comment:
0 is technically allowable, though rather suboptimal. Other aggregators also use `< 0` instead of `<= 0`, so I'm just following the pattern.